### PR TITLE
Add docs for info.json TAGS and REQUIREMENTS

### PR DESCRIPTION
**Pull request by calebj**

### CODE REVIEW

File: red/red_cog_info_json.md [hunk 1]
```diff
@@ -23,18 +23,26 @@ A repository info.json sits at the root level of the repository (Before the cog
...
 }
 ```
 
+The `REQUIREMENTS` key is used when installing the cog. Before it is loaded,
+the installer ensures that those packages are installed via `pip`.
+
+The `TAGS` key is used on our official cogs library portal, [cogs.red](https://cogs.red/)
+
 #### Info.json maker:
 [Link](https://gist.github.com/Twentysix26/73a2a09238d6875b5cb7)
```
Comment (Chovin) [inline red/red_cog_info_json.md]: looks good to me, but now the Info.json maker is out of date
shoud add a note or add the new sections to that as well before merge

**Comment by Chovin** [comment]
haven't tested, but looks good to me. 

If we're using this as a chance to update the whole thing in 1 PR..
[info_maker.py L44-46](https://gist.github.com/calebj/5623559dd1fc003623bedca8dc5f529b#file-info_maker-py-L44-L46)
iirc DISABLED is deprecated now. Will have to get @Twentysix26 to confirm.
Also just curious, does the order of the requirements ever matter?

**Comment by calebj** [comment]
Indeed, there is no use of `DISABLED` in downloader.py. I'll take it out.

The order shouldn't matter, no. Why?

**Comment by Chovin** [comment]
just curious in case people were relying on order

**Comment by Twentysix26** [comment]
We have an HIDDEN field now. Might want to include that.

**Pull request merged**